src/interfaces/IVotingMultipliers.sol

```diff
@@ -0,0 +1,35 @@
+pragma solidity ^0.8.22;
+
+import {IMultiplier} from "./multipliers/IMultiplier.sol";
+
+/// @title IVotingMultipliers
+/// @notice Interface for the VotingMultipliers contract
+/// @dev This interface defines the structure and functions for managing voting multipliers
+interface IVotingMultipliers {
+    /// @notice Thrown when attempting to add a multiplier that is already whitelisted
+    error MultiplierAlreadyWhitelisted();
```

Comment:
instead of "whitelist / blacklist", we should use "allowlist / denylist"

Reply:
addressed with 47333b3